packages/now-static-build/src/index.ts

```diff
@@ -160,7 +163,32 @@ function getPkg(entrypoint: string, workPath: string) {
   return pkg;
 }
 
-function getFramework(pkg: PackageJson) {
+function getFramework(config: Config, pkg?: PackageJson | null) {
+  if (config && config.framework && (config.framework as any).slug) {
```

Comment:
If think you can avoid all of the `as any` casts by starting with destructuring. Then all of the `if` statements will narrow down the types.

```js
const { framework, buildCommand, getOutputDirName, outputDirectory, distDir } = config;
```